src/librustc_codegen_ssa/back/linker.rs

```diff
@@ -1050,30 +1050,34 @@ impl<'a> Linker for WasmLd<'a> {
 }
 
 fn exported_symbols(tcx: TyCtxt, crate_type: CrateType) -> Vec<String> {
-    let mut symbols = Vec::new();
-
-    let export_threshold = symbol_export::crates_export_threshold(&[crate_type]);
-    for &(symbol, level) in tcx.exported_symbols(LOCAL_CRATE).iter() {
-        if level.is_below_threshold(export_threshold) {
-            symbols.push(symbol.symbol_name(tcx).to_string());
+    if let Some(ref exports) = tcx.sess.target.target.options.override_export_symbols {
+        exports.clone()
```

Comment:
Could this use a `return` to avoid the indentation of all the following lines?

Reply:
Certainly. The indentation actually bothered me a lot too.